accepted/00062-hub-xmlrpc-api.md

```diff
@@ -33,8 +33,11 @@ Content management APIs will work as of today. Inter-Server Synchronization (ISS
 
 ### New service and endpoint
 A new XMLRPC API endpoint will be created, implemented by a new service called the "XMLRPC Gateway API" (simply called "Gateway" from now on in this document). Technology-wise:
-  * Implementation will be in Python based on top of Tornado and [tornado-xmlrpc](https://pypi.org/project/tornado-xmlrpc/) library
-  * all I/O should be handled asynchronously via Tornado. Calls to several Servers should happen in parallel, with a maximum timeout value for unreachable/not responding Servers
+  * Implementation will be based on asynchronous I/O. Calls to several Servers should happen concurrently  
+  * The exact technology choice is delegated to the implementation phase and could be:
+    * typed Python, based on Tornado and [tornado-xmlrpc](https://pypi.org/project/tornado-xmlrpc/). Rationale: learning Tornado, which is also used in core Salt
```

Comment:
Actually, there are quite strong rumors in Salt community that asyncore is much more appealing option than Tornado (and it technically is better anyway). It is just not *yet* possible due to the necessity of Py2.x support and no official decision has been made yet. But as long as the minimal Python version stays on 3.6, very much chances are that Tornado actually might be dropped entirely. Especially taking to the account all the current austere issues with it in the Salt code. According to that,  https://github.com/python-trio/trio (which I myself as well like much more) is one of the candidates to replace Tornado. Ideally, if asyncore finally gets rid of event loop concept and just uses groups, similar to Go.

Thus I doubt if the rationale is strong enough here.

Reply:
Will update the text soon, thanks for this bit of information